Essentials/AutoCommand.cs

```diff
@@ -202,10 +208,22 @@ internal void RunNow()
 
         public override string ToString()
         {
-            return $"{Name} : {(Enabled ? "Enabled" : "Disabled")} : {Steps.Count}";
+            return $"{Name} : {_trigger.ToString()} : {Steps.Count}";
         }
     }
 
+    public enum Trigger
+    {
+        Disabled = -1,
```

Comment:
Enums always default to 0. Your default or disabled condition should always be 0, not -1. DayOfWeek uses -1 so the other values match with the System.DayOfWeek enum